src/plugins/betterGifAltText.ts

```diff
@@ -36,7 +36,7 @@ export default definePlugin({
         let name = url
             .slice(url.lastIndexOf("/") + 1)
             .replace(/\d/g, "") // strip numbers
-            .replace(/.gif$/, "") // strip extension
+            .replaceAll(/.gif/g, "") // strip extension
```

Comment:
is the point of this to strip `some-gif.gif.gif.gif`?
```suggestion
            .replaceAll(/(.gif)+$/, "") // strip extension
```

Reply:
huh? you want to replace `.gif.gif.gif.gif.gif.gif.gif.gif (infinite amount of .gif later)`?

Reply:
no its like, some things are `big red potato gif` and `.gif` would remove the .gif extension and the ` gif`

Reply:
it wouldn't. tenor links are aways `tag-tag-tag-gif-2129189.gif`, the `gif` is part of the tags

Reply:
so what i'd be removing is `-gif`

Reply:
I personally like having "gif" at the end of the alt text tbh

Reply:
![image](https://user-images.githubusercontent.com/53124886/193426991-4aa6fef7-62ae-4c5f-ab95-e5c77c433de4.png)
I doubt repeating the .gif pattern is necessary